code/modules/projectiles/guns/energy/rapidcrossbow.dm

```diff
@@ -1,14 +1,14 @@
-/obj/item/gun/energy/rxd 
+/obj/item/gun/energy/rxd
 	name = "Rapid Crossbow Device"
 	desc = "A hacked and heavily modified RCD. In exchange for losing versitility, it can flash forge its own ammunition using cell charges."
 	icon = 'icons/obj/guns/energy/poweredcrossbow.dmi'
 	icon_state = "rxb_empty"
 	item_state = "rxb_empty"
 	flags = CONDUCT
-	slot_flags = SLOT_BACK|SLOT_BELT|SLOT_HOLSTER 
+	slot_flags = SLOT_BACK|SLOT_BELT|SLOT_HOLSTER
 	wielded_item_state = "_doble"
 	damage_multiplier = 0.8
-	penetration_multiplier = 0.9
+	penetration_multiplier = -0.1
```

Comment:
The word multiplier implies multiplication, maybe change it to modifier in another PR.